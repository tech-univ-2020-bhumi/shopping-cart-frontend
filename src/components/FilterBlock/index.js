import React from 'react';
import propTypes from 'prop-types';
import * as styles from './index.module.css';
import FilterTabs from '../FilterTabs';

const FilterBlock = (props) => {
  const { testId } = props;
  return (
    <div className={styles.filter} data-testid={testId}>
      <h3>Filter Categories</h3>
      <FilterTabs />
    </div>
  );
};

FilterBlock.propTypes = {
  testId: propTypes.string.isRequired,
};

export default FilterBlock;
