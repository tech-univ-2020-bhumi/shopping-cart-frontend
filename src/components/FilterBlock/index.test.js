import React from 'react';
import { render } from '@testing-library/react';
import FilterBlock from './index';

describe('the FilterBlock component ', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<FilterBlock testId="test-filter-block" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
