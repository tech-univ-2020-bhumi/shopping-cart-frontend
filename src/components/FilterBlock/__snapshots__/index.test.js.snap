// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the FilterBlock component  should render correctly 1`] = `
<DocumentFragment>
  <div
    data-testid="test-filter-block"
  >
    <h3>
      Filter Categories
    </h3>
    <div>
      <button
        data-testid="filter-button"
        type="button"
      >
        All
      </button>
    </div>
  </div>
</DocumentFragment>
`;
